src/meter.c

```diff
@@ -86,11 +86,11 @@ static void meter_draw_cb(lv_event_t * e) {
 
     for (uint16_t i = 0; i < count; i++) {
         if (db <= noise_level) {
-            rect_dsc.bg_color = lv_color_hex(0x777777);
+            rect_dsc.bg_color = lv_color_hex(0x00CC00);
```

Comment:
Let it be configurable and default will remain as it was.
The same comment for color changes below.